src/display/base_factory.js

```diff
@@ -143,14 +143,18 @@ class BaseSVGFactory {
     }
   }
 
-  create(width, height) {
+  create(width, height, noDims = true) {
```

Comment:
This changes the default behaviour of these classes, which could very well cause issues elsewhere. Furthermore, having an optional parameter where `undefined` implies `true` feels slightly unfortunate.

With that in mind, can we please change this to e.g. `create(width, height, skipDims = false) {` instead (with suitable changes below)?

Reply:
My bad, I thought `false` and I wrote `true`.